src/components/common/PopupModal.tsx

```diff
@@ -0,0 +1,142 @@
+          display: 'flex',
+          alignItems: 'center',
+          justifyContent: 'center',
+        },
+        content: {
+          position: 'relative',
+          width: '90%',
+          maxWidth: '400px',
+          padding: '20px',
+          borderRadius: '10px',
+          border: 'none',
+          inset: 'unset',
+          zIndex: 1001,
+          backgroundColor: theme.colors.brand.white,
+        },
+      }}
+    >
+      <Typography
+        variant="h2"
+        fontSize="1.25rem"
+        fontWeight="bold"
+        marginBottom="1rem"
+        color={theme.colors.brand.primary}
+      >
+        {title}
+      </Typography>
+
+
+      <CloseButton
+        onClick={onClose}
+        style={{
+          position: 'absolute',
+          top: '14px',
+          right: '10px',
+          fontSize: '28px',
+          cursor: 'pointer',
+        }}
+        color={theme.colors.brand.primary}
+      >
+        &times;
+      </CloseButton>
+
+      {/* Message */}
+      <Typography
+        variant="p"
+        fontSize="14px"
+        marginBottom="10px"
+        color={theme.colors.brand.textSecondary}
+      >
+        {message}
+      </Typography>
+
+      <Box
+
+      >
+        <Box display="flex" justifyContent="center" mb={4}>
+          {['FF', 'LL', 'DD', 'MM', 'YYYY'].map((item) => (
+            <Box
+              key={item}
+              p="5px 10px"
+              bg={theme.colors.brand.backgroundGrey}
+              fontWeight="bold"
+              fontSize="14px"
+            >
+              {item}
+            </Box>
+          ))}
+        </Box>
+
+        <Box color={theme.colors.brand.textPrimary} textAlign="left">
+          <Typography variant="h2" color={theme.colors.brand.textSecondary} fontSize="14px" style={{ paddingLeft: '28px' }} ><strong>FF:</strong> {t("FIRST_TWO_LETTERS_OF_YOUR_FIRST_NAME")}</Typography>
+          <Typography variant="h2" color={theme.colors.brand.textSecondary} fontSize="14px" style={{ paddingLeft: '28px' }}><strong>LL:</strong>{t("FIRST_TWO_LETTERS_OF_YOUR_LAST_NAME")}</Typography>
+          <Typography variant="h2" color={theme.colors.brand.textSecondary} fontSize="14px" style={{ paddingLeft: '28px' }}><strong>DD:</strong> {t("DATE")} </Typography>
+          <Typography variant="h2" color={theme.colors.brand.textSecondary} fontSize="14px" style={{ paddingLeft: '28px' }}><strong>MM:</strong> {t("MONTH")} </Typography>
+          <Typography variant="h2" color={theme.colors.brand.textSecondary} fontSize="14px" style={{ paddingLeft: '28px' }}><strong>YYYY:</strong> {t("YEAR")} </Typography>
+        </Box>
+
+      </Box>
+
+      {/* Example */}
+      <h2 style={{ color: theme.colors.brand.textSecondary, fontSize: '12px', marginTop: '10px', marginBottom: '10px', backgroundColor: theme.colors.brand.backgroundHighlight, padding: '10px' }}>
+        <strong>{t("EXAMPLE")}:</strong><i>{example}</i> <strong>‘ANKU30121988’</strong>
+      </h2>
```

Comment:
The Typography component is not a single component; it should include H1, H2, ..., H6, etc.